iOS Client/PodcastsController/MBPodcastsViewController.swift

```diff
@@ -25,6 +25,7 @@ class MBPodcastsViewController: UIViewController, UITableViewDataSource, UITable
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        self.navigationController?.navigationBar.isTranslucent = false
```

Comment:
does this belong in the Podcast's coordinator's start() function or the nav controller lazy property code? Curious about your thoughts. 

I've been altering the navigation Item of the current view controller in the view controller, which pretty much assumes knowledge of the containing nav controller.

This goes one step further, and actually alters a property on the parent nav controller's nav bar.

I actually think it's okay if it remains here after thinking about it, b/c it has to do with appearance and won't crash if someone were to drop this VC in a context where it is not embedded in a nav controller.

Reply:
yeah this needs to be here because when coming back from podcast detail the nav bar needs to change from translucent to non translucent